.github/ISSUE_TEMPLATE/improve-the-site.yml

```diff
@@ -0,0 +1,42 @@
+name: Improve the docs.getdbt.com site
+description: Make a suggestion or report a problem about the technical implementation of docs.getdbt.com.
+labels: ["engineering"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
+        * Before you file an issue read the [Contributing guide](https://github.com/dbt-labs/docs.getdbt.com#contributing).
+        * Check to make sure someone hasn't already opened a similar [issue](https://github.com/dbt-labs/docs.getdbt.com/issues).
+
+  - type: checkboxes
+    id: contributions
+    attributes:
+      label: Contributions
+      description: Please read the contribution docs before opening an issue or pull request.
+      options:
+        - label: I have read the contribution docs, and understand what's expected of me. 
+
+  - type: textarea
+    attributes:
+      label: What article(s) or areas on docs.getdbt.com are affected?
+      description: Include links to articles where you're seeing a problem, screenshots, what browser you're using, etc.
```

Comment:
Is this just for reporting a problem or might they also be making a suggestion about change?

Reply:
I like the idea of keeping it open to either!